src/Combobox/ComboboxItem.tsx

```diff
@@ -1,7 +1,27 @@
 import React from "react";
 import PropTypes from "prop-types";
```

Comment:
what do you think about removing prop-types from components converted to TS?

Reply:
I agree, but I want to do that in a separate step to ensure we don't lose the args tables in storybook documentation as we convert over